fe/src/components/IssueTable/IssueTable.jsx

```diff
@@ -1,51 +1,108 @@
 import styled from 'styled-components';
+import { useContext } from 'react';
 import CheckBox from './CheckBox/CheckBox';
 import IssueItem from './IssueItem/IssueItem';
 import IssueStatusButtons from './TableToolBar/IssueStatusButtons/IssueStatusButtons';
 import TableFilterButtons from './TableToolBar/TableFilterButtons/TableFilterButtons';
 import TableToolBar from './TableToolBar/TableToolBar';
+import {
+  MainPageContext,
+  MainPageDispatchContext,
+  mainPageInitialState,
+} from '../../context/MainPage/MainPageContext';
+import { Button } from '../common';
+import { OPENED, RESET } from '../../constants';
+import { ReactComponent as xSquare } from '../../assets/xSquare.svg';
+import { setFilterOption } from '../../context/MainPage/MainPageActions';
 
-const IssueTable = ({
-  issues, counts, issueState, setFilterOptions,
-}) => {
-  const issueItems = issues?.length ? (
-    issues.map((issue) => <IssueItem key={issue.number} {...issue} />)
-  ) : (
-    <NoticeBox>검색과 일치하는 결과가 없습니다.</NoticeBox>
+const IssueTable = () => {
+  const { issues, filterOptions, loading } = useContext(MainPageContext);
+  const dispatch = useContext(MainPageDispatchContext);
+
+  let content;
+
+  if (loading) {
+    // TODO(덴): 스피너 기능에 대한 고민 필요. 굳이 필요한가?
+    // content = <Spinner />;
+  } else if (issues.length) {
+    content = issues.map((issue) => (
+      <IssueItem key={issue.number} {...issue} />
+    ));
+  } else {
+    content = <NoticeBox>검색과 일치하는 결과가 없습니다.</NoticeBox>;
+  }
```

Comment:
아 이부분은 이렇게 변수로 빼는 처리를 하셨군요. 지난번의 리뷰지점이어서 이렇게 변경된 것으로 보여져요.
이 부분은 code suggestion을 드려보겠습니다.
```suggestion
const issueItems = issues && issues.length !== 0 && (
    issues.map((issue) => <IssueItem key={issue.number} {...issue} />)
  ) 
```